components/compliance-service/api/tests/25_nodes_spec.rb

```diff
@@ -201,6 +206,7 @@ def cleanup
           "id": node_id1,
           "name": "M$",
           "manager": "automate",
+          "tags": [{}],
```

Comment:
the switch to json_build_object with json_agg leaves us with an empty array with an empty object inside. this isn't really problematic, but im open to suggestions on how to make it not happen..

Reply:
This should be fixed with the below SQL change.